script.js

```diff
@@ -6,7 +6,25 @@ function setup() {
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  return episodeList.map(episode => {
+    const div = document.createElement("div");
+    div.classList.add("episode");
+
+    const h3 = document.createElement("h3");
+    h3.textContent = `${episode.name} - ${episode.number < 10
+      ? "S0"+episode.season+"E0"+episode.number 
+      : "S0"+episode.season+"E"+episode.number}`;
+
```

Comment:
- please make a separate function and put there the logic that makes the title out of episode's object.
- use `String.padStart()` instead of that ternary operator